params.yml

```
@@ -172,4 +172,4 @@ args:
   ## Permissionless node configuration.
   trusted_sequencer_node_uri: zkevm-node-sequencer-001:6900
   zkevm_aggregator_host: zkevm-node-aggregator-001
-  genesis_file: templates/permissionless-node/genesis.json
+  genesis_file: templates/permissionless-node/genesis.json
```

Nitpick, new line missing?

```suggestion
  genesis_file: templates/permissionless-node/genesis.json
  
```